### PR TITLE
add rbac plugins and example playbook

### PR DESCRIPTION
**Pull request by tphee**

### CODE REVIEW

File: plugins/modules/managed_serviceaccount_addon.py [hunk 1]
```diff
@@ -347,7 +274,13 @@ def execute_module(module: AnsibleModule):
     # get token
     token_bytes = base64.b64decode(secret.data.token)
     token = token_bytes.decode('ascii')
-    module.exit_json(token=token)
+    managed_serviceaccount = {
+        'name': managed_service_account.metadata.name, 'namespace': managed_service_account.metadata.namespace,
+        'api_version': managed_service_account.apiVersion, 'kind': managed_service_account.kind,
+        'uid': managed_service_account.metadata.uid, 'service_account': {'name': managed_service_account.metadata.name,
+                                                                         'namespace': managed_service_account_addon.spec.installNamespace, 'token': token}
```
Comment (TheRealHaoLiu) [inline plugins/modules/managed_serviceaccount_addon.py:281]: we can probably simplify the output a little to scope to the information we need to expose to the user
```             
{
  "name": "name of the ManagedServiceAccount resource",
  "namespace": "namespace of the ManagedServiceAccount resource", //cluster-namespace
  "token": "token of the ServiceAccount"
}
```
we need to examine how do we do deletion of managed-serviceaccount in "ansible style"

File: plugins/modules/managed_serviceaccount_rbac_addon.py [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+    except Exception:
+        module.fail_json(
+            msg=f"error: invalid RBAC template file {filename}")
+
+    manifest_work_api = hub_client.resources.get(
+        api_version='work.open-cluster-management.io/v1',
+        kind='ManifestWork',
+    )
+
+    manifest_work = manifest_work_api.create(new_manifest_work)
```
Comment (TheRealHaoLiu) [inline plugins/modules/managed_serviceaccount_rbac_addon.py:202]: we may want to wait for manifestwork to complete applying

Comment (tphee) [reply plugins/modules/managed_serviceaccount_rbac_addon.py:202]: The wait is already there:

```
   if wait:
        wait_for_manifestwork_available(
            module, hub_client, manifest_work, timeout)
```

**Review by hanqiuzh — APPROVED** [review]

**Pull request merged**